src/main/cpp/include/loader/tiledb_loader_file_base.h

```diff
@@ -123,12 +147,8 @@ class BufferReaderBase : public virtual GenomicsDBImportReaderBase {
     if (src == 0)
       return 0u;
     if (m_num_valid_bytes_in_buffer+num_bytes > m_buffer.size()) {
-#ifdef DEBUG
-      std::cerr << "Buffer resized from " << m_buffer.size()
-                << " bytes to " <<  m_num_valid_bytes_in_buffer+num_bytes << "\n";
-#endif
+      logger.debug("Buffer resized from {} bytes to {}", m_buffer.size(), m_num_valid_bytes_in_buffer+num_bytes);
```

Comment:
@kgururaj, I have a `logger.debug_only` api too, should this be changed to using that? With logger.debug_only, changing the logging level will have no effect if the library was built as a release. But, logger.debug can be turned on/off with logging level.